submissions/ArthurGorbenko/html.css.practice/index.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+
+<title>Html/css practice</title>    
+
+<link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main>
+        <input type="checkbox" id="trig" name="trigger">
+        <nav class="navbar">
+            <div class="container">
```

Comment:
For navigation item better use list. It helps to screenreader to recognize the full structure of the menu list.

Reply:
> To be honest, i know about that behavior, i tried to fix. There is a thing: i need to declare input before ul that we need to hide. In case of using display:flex;flex-direction: column-reverse;( only with this we can get affect like autoscroll to the bottom of a list), only one place afford for input.But i am open for tricks and advises.
I had already conversation about this.If i replace all markup with ul, i 'll need wrap and `input` tag also, but it will cause unworking menu.

Reply:
I don't understand what you mean: ' it will cause unworking menu'. Watch your markup and everything will work.

Reply:
If i right understood, u meaned use `ul` list for that puprpose.This is my [attempt](https://github.com/ArthurGorbenko/Html_css_popup/commit/83731cf83c75594a99acc9a38f1c2e680b966ab7)

Reply:
Looks good, but:
-  move `input.trig` to the `li.pointsMenu`
-  put all content from `li.content` to the `li.pointsMenu`

Your menu list must contain only 5 items. Now you have 6 menu items and one `input`.
Try to use more meaningful class names instead of `.trig`.

Reply:
[Did](https://github.com/ArthurGorbenko/Html_css_popup/commit/912adc7cb63c4260e0125f879aea028c38862043) movement `input` and `div.content`.Also [renamed](https://github.com/ArthurGorbenko/Html_css_popup/commit/bc5924732760f8d30bc2c061a454fd34f32df123) class for `input`.